remote/Dockerfile

```diff
@@ -45,6 +45,24 @@ RUN wget -qO /usr/share/keyrings/xpra.asc ${XPRA_REGISTRY}/xpra.asc && \
         nginx && \
     rm -rf /var/lib/apt/lists/*
 
+# Inject script to force focus on modal windows in xpra client
+RUN sed -i '/load_default_settings();/a \
+setInterval(() => {\
+for (const [id, win] of Object.entries(client.id_to_window).toReversed()) {\
+    if (win.metadata.modal) {\
+        if (client.topwindow !== id) {\
+            client.set_focus(win);\
+            break;\
+        }\
+    }\
+}\
+}, 100);' /usr/share/xpra/www/index.html
+
+# Remove the compressed index.html files because they don't contain our fix
+RUN rm /usr/share/xpra/www/index.html.gz
+RUN rm /usr/share/xpra/www/index.html.br
+
+
```

Comment:
```suggestion
# Inject script to force focus on modal windows in xpra client
# and remove the compressed index.html files because they don't contain our fix
RUN sed -i '/load_default_settings();/a \
setInterval(() => {\
for (const [id, win] of Object.entries(client.id_to_window).toReversed()) {\
    if (win.metadata.modal) {\
        if (client.topwindow !== id) {\
            client.set_focus(win);\
            break;\
        }\
    }\
}\
}, 100);' /usr/share/xpra/www/index.html \
&& rm /usr/share/xpra/www/index.html.gz \
&& rm /usr/share/xpra/www/index.html.br
```

To reduce the number of layers, we should merge the RUN statements into one.